app/controllers/contacts_controller.rb

```diff
@@ -1,19 +1,14 @@
 # frozen_string_literal: true
 
 class ContactsController < ApplicationController
-  def search
+  def index
     @edition = Edition.find_current(document: params[:document])
     assert_edition_state(@edition, &:editable?)
     assert_edition_access(@edition, current_user)
     @contacts_by_organisation = ContactsService.new.all_by_organisation
+    render layout: rendering_context
   rescue GdsApi::BaseError => e
     GovukError.notify(e)
-    render "search_api_down", status: :service_unavailable
-  end
-
-  def insert
-    Contacts::InsertInteractor.call(params: params, user: current_user)
-
-    redirect_to edit_document_path(params[:document], anchor: "body")
+    render "index_api_down", layout: rendering_context
```

Comment:
I think we just need to modify the existing API down feature to cover this, and we're there 🎆 .

Reply:
Yup, I've split our `*_api_down` test into `*_spec` and `*_no_js_spec` testing both scenarios.